Singleton/AccountCounter.cs

```diff
@@ -1,16 +1,32 @@
 ﻿using System;
+using System.IO;
 using System.Collections.Generic;
 using System.Text;
 
 namespace Singleton
 {
-    public sealed class AccountCounter
+    public sealed class LoggerSingleton
     {
-        private static readonly Lazy<AccountCounter> lazy = new Lazy<AccountCounter>(() => new AccountCounter());
+        private static readonly Lazy<LoggerSingleton> instance = new Lazy<LoggerSingleton>(() => new LoggerSingleton());
+        private int logID = 1;
+        private StreamWriter logger;
 
-        public static AccountCounter Instance { get { return lazy.Value; } }
-        public int count = 0;
+        private LoggerSingleton()
+        {
+            logger = new StreamWriter("logger.txt");
+        }
 
-        private AccountCounter() { }
+        public void WriteInLog(string logMessage)
+        {
+            DateTime now = DateTime.Now;
+            logger.Write("Log ID: " + logID + " Date:"  + now + " " + logMessage);
+        }
+
+        public void FinishLogger()
```

Comment:
Itæs better to implement Disposable pattern here: https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
I've used "using" section for Logger logic, is it the same?